### PR TITLE
[W29] evanxd: 07/18 - 07/22

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

File: evanxd.md [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
-* [SensorWeb]
-  - Bug 1286142 - Update contributor list
+    - Helped Taipei Dev Tools members understand how to use Rep Tester
+      - Currently Taipei Dev Tools members can use Rep Tester.
+    - Bug 1282791 - Reps: uninteresting props algorithm is wrong, review+
+    - Bug 1276376 - Reps: uninteresting props are not filtered correctly
+      - Fixed this in Bug 1282791
+    - Bug 1284838 - Reps: render events more uniformly, feedback?
 
 ### This week
```
Comment (timdream) [inline evanxd.md]: Can you drop the last week - this week format? We have not been using this (and rely on commit diff for a long time). Thanks.

**Comment by timdream** [comment]
Can you please also work on https://bugzilla.mozilla.org/show_bug.cgi?id=1277105 at the same time w/ other DevTools work? It might be as easy as verify this is already fixed somewhere, bot hasn't comment on this for 14 days.

**Comment by timdream** [comment]
Thank you for making your progress on DevTools!

File: evanxd.md (hunk 1 above)
Comment (evanxd) [inline evanxd.md]: Got it. Thanks.

**Comment by evanxd** [comment]
Sure, I'll work on https://bugzilla.mozilla.org/show_bug.cgi?id=1277105 this week.

File: evanxd.md (hunk 1 above)
Comment (evanxd) [inline evanxd.md]: Sorry, didn't see the commands.

**Pull request merged**